soak/prow/scripts/soak-on-release.sh

```diff
@@ -138,8 +138,10 @@ helm install $SOAK_CHART_RELEASE_NAME . \
 # Loop until the Job executing soak test does not complete. Check again with 30 minutes interval.
 while kubectl get jobs/$AWS_SERVICE-soak-test -o=json | jq -r --exit-status '.status.completionTime'>/dev/null; [ $? -ne 0 ]
 do
-  >&2 echo "soak-on-release.sh] [INFO] Completion time is not present in the job status. Soak test is still running."
-  >&2 echo "soak-on-release.sh] [INFO] Sleeping for 30 mins..."
+  >&2 echo "soak-on-release.sh] [INFO] Current soak Job(default/$AWS_SERVICE-soak-test) status is: "
+  kubectl get jobs/$AWS_SERVICE-soak-test -o=json | jq -r '.status' >&2
+  >&2 echo "soak-on-release.sh] [INFO] Completion time is not present in the soak Job status. Soak Job is still running."
+  >&2 echo "soak-on-release.sh] [INFO] Current time is $(date) . Sleeping for 30 mins ..."
```

Comment:
Why `echo` to `stderr`?

Reply:
Funny you ask. I learnt it from your code of [wrapper.sh](https://github.com/aws-controllers-k8s/test-infra/blob/main/prow/jobs/images/wrapper.sh) 

I assumed it is because if stdout is not printed in some systems to avoid noise but stderr is much more important and would be printed. Please correct me if there is a better reason for doing ">&2".

Reply:
`wrapper.sh` was taken from upstream K8s and I don't particularly understand why they print everything to both pipes. For Prow the stdout shows in the logs anyway, so it doesn't make any difference.

Reply:
Okay, I will remove it.